docs/book/toc.md

```diff
@@ -154,7 +154,7 @@
 
 ## Reference
 
-* [🤷 SDK & CLI reference](https://apidocs.zenml.io/)
+* [🤷 SDK & CLI reference](https://apidocs.zenml.io/0.40.3/)
```

Comment:
@AlexejPenner Is this needed or will it be automatically converted by your script on release? On the `release/0.41.0` it didn't get the version suffix from what I can see